tests/tasks/test_dbus.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2015, Facebook, Inc.  All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+from sparts.tests.base import MultiTaskTestCase, Skip
+try:
+    import dbus
+    from sparts.tasks.dbus import DBusServiceTask, DBusMainLoopTask
+except ImportError:
+    raise Skip("dbus support is required to run this test")
+
+from sparts.sparts import option
+
+class TestDBusTask(DBusServiceTask):
+    BUS_NAME = 'com.github.facebook.test'
+
+
+class TestDBusSystemTask(DBusServiceTask):
+    BUS_NAME = 'com.github.facebook.systemtest'
+
+    system_bus = option(name='system_bus', default=True)
+
+    def start(self):
+        try:
+            super(TestDBusSystemTask, self).start()
+        except dbus.DBusException as err:
+            self.acquire_name_error = str(err)
+
+
+class TestDBus(MultiTaskTestCase):
+    TASKS = [TestDBusTask, DBusMainLoopTask]
+
+    def test_session_bus(self):
+        bus = dbus.SessionBus(private=True)
+        self.assertTrue(bus.name_has_owner(TestDBusTask.BUS_NAME))
```

Comment:
Will this test fail if two tests are run at the same time (and try to grab the same bus name from the SessionBus)?

Might bet worth appending a random 64-bit number + timestamp to the end of the `BUS_NAME` to avoid this.